tests/framework/installer/ceph_csi_templates.go

```diff
@@ -137,9 +137,9 @@ const (
               hostPath:
                 path: /lib/modules
             - name: socket-dir
-              hostPath:
-                path: /var/lib/kubelet/plugins/{{ .DriverNamePrefix }}rbd.csi.ceph.com
-                type: DirectoryOrCreate
+              emptyDir: {
```

Comment:
this template is used for v1.0 not for master, to get things going I filed a separate PR https://github.com/rook/rook/pull/4650

Reply:
@Madhu-1 the suggestion by @eparis was to not fiddle with the priv/non-priv nature of the containers but have a first PR that does only the emptydir patch. The theory is that this would suffice to fix the permission issue.

Can you explain why you again combine it with a patch that changes some containers to privileged?

Reply:
@Madhu-1 are you saying that the extra patch to `tests/framework/installer/ceph_csi_templates.go` is unneeded and even wrong? In that case I can just update this PR to remove it and have only your patch.

Reply:
> @Madhu-1 are you saying that the extra patch to `tests/framework/installer/ceph_csi_templates.go` is unneeded and even wrong? In that case I can just update this PR to remove it and have only your patch.

These templates are used for upgrade testing from v1.0 , and in v1.0 rook has disabled the CSI deployment, we should not make any changes to theses files, @travisn can you please confirm